tensorflow/core/kernels/mkl/mkl_matmul_op_benchmark.cc

```diff
@@ -29,9 +32,37 @@ static Graph* Matmul(int m, int k, int n, bool transpose_a, bool transpose_b,
   in0.flat<T>().setRandom();
   Tensor in1(type, transpose_b ? TensorShape({n, k}) : TensorShape({k, n}));
   in1.flat<T>().setRandom();
-  test::graph::oneDNNMatmul(g, test::graph::Constant(g, in0),
-                            test::graph::Constant(g, in1), transpose_a,
-                            transpose_b);
+
+  Node* src0 = test::graph::Constant(g, in0);
+  Node* src1 = test::graph::Constant(g, in1);
+  g->AddEdge(g->source_node(), 0, src0, 0);
+  g->AddEdge(g->source_node(), 1, src1, 0);
+  // Add shape sizes
+  AttrValue attr_input_shape;
+  TensorShapeProto* proto = attr_input_shape.mutable_list()->add_shape();
+  proto->add_dim()->set_size(m);
+  proto->add_dim()->set_size(k);
+  proto = attr_input_shape.mutable_list()->add_shape();
+  proto->add_dim()->set_size(k);
+  proto->add_dim()->set_size(n);
+
+  Node* ret = nullptr;
+
+  TF_CHECK_OK(NodeBuilder(g->NewName("matmul"), "MatMul")
+                  .Input(src0)
+                  .Input(src1)
+                  .Attr("transpose_a", transpose_a)
+                  .Attr("transpose_b", transpose_b)
+                  .Attr("_input_shapes", attr_input_shape)
+                  .Finalize(g, &ret));
+  int ret_id = ret->id();
+  if (IsMKLEnabled()) {
+    std::unique_ptr<Graph>* ug = new std::unique_ptr<Graph>(g);
+    RunMklLayoutRewritePass(ug);
+    if (!g->FindNodeId(ret_id)) {
+      ret = g->FindNodeId(ret_id + 1);
+    }
```

Comment:
Why do we need this `if` when we don't use the value of `ret` anymore?

Reply:
I have aligned this change with what we did in https://github.com/tensorflow/tensorflow/pull/60160/files#diff-99dd0db5515ffbb4eada93728cba4daa7d6f27405ef63114b8674d4e7def3367R125 and removed the unused code.